packages/@sanity/google-maps-input/src/loader/LoadError.tsx

```diff
@@ -17,7 +17,7 @@ export function LoadError(props: Props) {
           <>
             <h3>Error details:</h3>
             <pre>
-              <code>{props.error.message}</code>
+              <code>{props.error?.message}</code>
```

Comment:
This is just TypeScript not being clever enough to know that `props.error` must be defined in this branch, so I believe you can safely use `props.error!.message` here instead.